front/app/modules/commercial/moderation/admin/containers/index.tsx

```diff
@@ -419,16 +404,25 @@ const Moderation = memo<Props & InjectedIntlProps>(({ className, intl }) => {
     }
   }, [list, processing]);
 
-  const selectedModerationItemsWithContentWarning = useMemo(() => {
-    if (!isNilOrError(moderationItems)) {
-      return moderationItems.filter(
-        (moderationItem) =>
-          moderationItem.relationships.inappropriate_content_flag
-      );
-    }
-
-    return null;
-  }, [moderationItems]);
+  useEffect(() => {
+    (async () => {
+      if (selectedModerations.length > 0) {
+        const selectedModerationsWithFlagRelationship = selectedModerations.filter(
+          (moderation) =>
+            moderation.relationships.inappropriate_content_flag?.data.id
+        );
+        const promises = selectedModerationsWithFlagRelationship.map(
+          (moderationItemWithFlag) =>
+            inappropriateContentFlagByIdStream(
+              moderationItemWithFlag.id
+            ).fetch()
+        ) as Promise<IInappropriateContentFlag>[];
+        const flags = await Promise.all(promises);
```

Comment:
you know it... `try catch`? :)

Reply:
Here, I wouldn't do it as it's not a user action so an error would be a bit strange, no? :)